docs/source/tensors.rst

```diff
@@ -8,21 +8,34 @@ torch.Tensor
 A :class:`torch.Tensor` is a multi-dimensional matrix containing elements of
 a single data type.
 
-Torch defines nine CPU tensor types and nine GPU tensor types:
+Torch defines 10 tensor types with CPU and GPU variants:
 
-========================   ===========================================   ===========================   ================================
+========================== ===========================================   ============================= ================================
 Data type                  dtype                                         CPU tensor                    GPU tensor
-========================   ===========================================   ===========================   ================================
+========================== ===========================================   ============================= ================================
 32-bit floating point      ``torch.float32`` or ``torch.float``          :class:`torch.FloatTensor`    :class:`torch.cuda.FloatTensor`
 64-bit floating point      ``torch.float64`` or ``torch.double``         :class:`torch.DoubleTensor`   :class:`torch.cuda.DoubleTensor`
-16-bit floating point      ``torch.float16`` or ``torch.half``           :class:`torch.HalfTensor`     :class:`torch.cuda.HalfTensor`
+16-bit floating point [1]_ ``torch.float16`` or ``torch.half``           :class:`torch.HalfTensor`     :class:`torch.cuda.HalfTensor`
+16-bit floating point [2]_ ``torch.bfloat16``                            :class:`torch.BFloat16Tensor` :class:`torch.cuda.BFloat16Tensor`
+32-bit complex             ``torch.complex32``
+64-bit complex             ``torch.complex64``
+128-bit complex            ``torch.complex128`` or ``torch.cdouble``
 8-bit integer (unsigned)   ``torch.uint8``                               :class:`torch.ByteTensor`     :class:`torch.cuda.ByteTensor`
 8-bit integer (signed)     ``torch.int8``                                :class:`torch.CharTensor`     :class:`torch.cuda.CharTensor`
 16-bit integer (signed)    ``torch.int16`` or ``torch.short``            :class:`torch.ShortTensor`    :class:`torch.cuda.ShortTensor`
 32-bit integer (signed)    ``torch.int32`` or ``torch.int``              :class:`torch.IntTensor`      :class:`torch.cuda.IntTensor`
 64-bit integer (signed)    ``torch.int64`` or ``torch.long``             :class:`torch.LongTensor`     :class:`torch.cuda.LongTensor`
 Boolean                    ``torch.bool``                                :class:`torch.BoolTensor`     :class:`torch.cuda.BoolTensor`
-========================   ===========================================   ===========================   ================================
+========================== ===========================================   ============================= ================================
+
+.. [1] 
+  Sometimes referred to as binary16: uses 1 sign, 5 exponent, and 9 fraction
```

Comment:
10 mantissa bits (7 mantissa bits for bfloat16 below)

Reply:
Sorry, somehow missed this even though I thought I fixed it. Should be fixed in a335e16